exoplayer-flac/README.md

```diff
@@ -34,27 +34,27 @@ FLAC_EXT_PATH="${EXOPLAYER_ROOT}/extensions/flac/src/main"
 NDK_PATH="<path to Android NDK>"
 ```
 
-* Download and extract flac-1.3.1 as "${FLAC_EXT_PATH}/jni/flac" folder:
+* Download and extract flac-1.3.2 as "${FLAC_EXT_PATH}/jni/flac" folder:
 
 ```
 cd "${FLAC_EXT_PATH}/jni" && \
-curl https://ftp.osuosl.org/pub/xiph/releases/flac/flac-1.3.1.tar.xz | tar xJ && \
-mv flac-1.3.1 flac
+curl https://ftp.osuosl.org/pub/xiph/releases/flac/flac-1.3.2.tar.xz | tar xJ && \
+mv flac-1.3.2 flac
 ```
 
 * Build the JNI native libraries from the command line:
 
 ```
 cd "${FLAC_EXT_PATH}"/jni && \
-${NDK_PATH}/ndk-build APP_ABI=all -j4
+${NDK_PATH}/ndk-build.cmd APP_ABI=all -j4
 ```
 
 [top level README]: https://github.com/google/ExoPlayer/blob/release-v2/README.md
 [Android NDK]: https://developer.android.com/tools/sdk/ndk/index.html
 
 ## Using the extension ##
 
-Once you've followed the instructions above to check out, build and depend on
+Once you've followed the instructions above to check out, build, and depend on
```

Comment:
Did you pull in changes from the ExoPlayer repo from [`ExoPlayer/extensions/flac`](https://github.com/google/ExoPlayer/tree/release-v2/extensions/flac)? Because that's where all of this is from.

I'd prefer to keep this in-sync with the original repo, even if there are grammatical errors. Do you mind reverting the changes to the `exoplayer-flac` module?